SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes.Android/ErrorReport.cs

```diff
@@ -23,15 +22,14 @@ internal ErrorReport(AndroidErrorReport androidReport)
             {
                 MobileCenterLog.Debug(Crashes.LogTag, "Cannot read throwable from java point of view, probably a .NET exception", e);
                 androidThrowable = null;
-                byte[] exceptionBytes = AndroidExceptionDataManager.LoadWrapperExceptionData(Java.Util.UUID.FromString(Id));
-                if (exceptionBytes != null)
-                {
-                    Exception = CrashesUtils.DeserializeException(exceptionBytes);
-                }
             }
-
             AndroidDetails = new AndroidErrorDetails(androidThrowable, androidReport.ThreadName);
             iOSDetails = null;
+            byte[] exceptionBytes = AndroidExceptionDataManager.LoadWrapperExceptionData(Java.Util.UUID.FromString(Id));
+            if (exceptionBytes != null)
+            {
+                Exception = CrashesUtils.DeserializeException(exceptionBytes);
```

Comment:
`Exception = CrashesUtils.DeserializeException(exceptionBytes)` looks weird here. Should it be a lowercase `exception` variable name here? If you decide to rename this, please also fix the one in file [Microsoft.Azure.Mobile.Crashes.iOS/ErrorReport.cs](https://github.com/Microsoft/mobile-center-sdk-dotnet/blob/3f18680bb84761896639aad5c8c80d0954336eb3/SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes.iOS/ErrorReport.cs#L33).